test/test_run/test_integration.py

```diff
@@ -214,20 +243,26 @@ def make_empty_pdf(*args, **kwargs):
 
 @pytest.fixture
 def get_config_path():
-    def do_get_config_path(rmd=False):
-        dirname = "autograder" if not rmd else "rmd-autograder"
+    def do_get_config_path(rmd=False, qmd=False):
+        #dirname = "autograder" if not rmd else "rmd-autograder"
+        if rmd == False & qmd == False:
+            dirname = "autograder"
+        if rmd == True & qmd == False:
+            dirname = "rmd-autograder"
+        if rmd == False & qmd == True:
+            dirname = "rmd-autograder"
         return FILE_MANAGER.get_path(f"{dirname}/source/otter_config.json")
     return do_get_config_path
 
-
 @pytest.fixture
 def load_config(get_config_path):
-    def load_config_file(rmd=False):
-        with open(get_config_path(rmd=rmd)) as f:
+    def load_config_file(rmd=False, qmd=False):
+        with open(get_config_path(rmd=rmd, qmd=False)) as f:
+            return json.load(f)
+        with open(get_config_path(rmd=False, qmd=qmd)) as f:
```

Comment:
You don't need multiple `with` blocks; you can just do a single one with `with open(get_config_path(rmd=rmd, qmd=qmd)) as f:`